diff --git a/dist/chunks/rpc.-pEldfrD.js b/dist/chunks/rpc.-pEldfrD.js
index a48eb0174a4fbc508a36c46ff78b0f9bf7ba6295..cfac96842755712ab806b8a84170ddaad7c715bf 100644
--- a/dist/chunks/rpc.-pEldfrD.js
+++ b/dist/chunks/rpc.-pEldfrD.js
@@ -48,7 +48,1 @@ function createRuntimeRpc(options) {
-		onTimeoutError(functionName, args) {
-			let message = `[vitest-worker]: Timeout calling "${functionName}"`;
-			if (functionName === "fetch" || functionName === "transform" || functionName === "resolveId") message += ` with "${JSON.stringify(args)}"`;
-			// JSON.stringify cannot serialize Error instances
-			if (functionName === "onUnhandledError") message += ` with "${args[0]?.message || args[0]}"`;
-			throw new Error(message);
-		},
+		timeout: -1,
